### PR TITLE
Convert MainActivityFragment to kotlin

### PR DESCRIPTION
**Pull request by thusson13**
https://github.com/wayfair/brickkit-android/issues/204

### CODE REVIEW

File: BrickKit/app/build.gradle [hunk 1]
```diff
@@ -33,6 +35,7 @@ android {
 }
 
 dependencies {
+    implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.72'
```
Comment (kenyee) [inline BrickKit/app/build.gradle]: you don't need this w/ Kotlin 1.4... also shouldn't be using 1.3.72 any more..

File: BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+class MainActivityFragment : BrickFragment() {
+    private var rowNumber = 0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val placeholderBrick = ActiveBrick(resources, R.string.empty) { changeFragment(SimpleBrickFragment()) }
+            .also {
+                // Fake waiting as in a API request to show off placeholder when data is not ready
+                Handler(Looper.getMainLooper()).postDelayed(
```
Comment (kenyee) [inline BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt:36]: this is weird..why not just use Rx? 🤔

File: BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    { it.text = getString(R.string.simple_brick_title) },
+                    2000
```
Comment (kenyee) [inline BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt]: Should use something like TimeUnit.SECONDS.toMillis(2) I think?

File: BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt (hunk 2 above)
Comment (thusson13) [reply BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt:36]: I don't think we included rx in the brickkit repo so far.

File: BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt (hunk 3 above)
Comment (thusson13) [reply BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt]: done

File: BrickKit/app/build.gradle (hunk 1 above)
Comment (thusson13) [reply BrickKit/app/build.gradle]: done

**Review by mindwalkr — APPROVED** [review]

**Review by mindwalkr — APPROVED** [review]

**Pull request merged**